src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -115,6 +113,13 @@ public void run() {
             throwIfCancelled();
             updateMetadata(module, outputFile);
             throwIfCancelled();
+            for (Converter converter : converters) {
+                Path outputPath = converter.convert(outputFile);
```

Comment:
So Ideally you want to:
 * Create a temporary file
 * Call `tmpFile = converter.convert(outputFile)`
 * Move `tmpFile` => `outputFile`
 * Keep going with the loop